src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/ReindexJobTask.cs

```diff
@@ -190,7 +190,7 @@ private async Task<bool> TryPopulateNewJobFields(CancellationToken cancellationT
             // Find search parameters not in a final state such as supported, pendingDelete, pendingDisable.
             List<SearchParameterStatus> validStatus = new List<SearchParameterStatus>() { SearchParameterStatus.Supported, SearchParameterStatus.PendingDelete, SearchParameterStatus.PendingDisable };
             var searchParamStatusCollection = await _searchParameterStatusManager.GetAllSearchParameterStatus(cancellationToken);
-            var possibleNotYetIndexedParams = _searchParameterDefinitionManager.AllSearchParameters.Where(sp => validStatus.Contains(searchParamStatusCollection.First(p => p.Uri == sp.Url).Status) || sp.IsSearchable == false || sp.SortStatus == SortParameterStatus.Supported);
+            var possibleNotYetIndexedParams = _searchParameterDefinitionManager.AllSearchParameters.Where(sp => validStatus.Contains(searchParamStatusCollection.First(p => p.Uri == sp.Url).Status) || sp.IsSearchable == true || sp.SortStatus == SortParameterStatus.Supported);
```

Comment:
I am not sure if the change to sp.IsSearchable == true is correct.  It seems that now the overall logic is:

Where:
1) any status is one of the three validStatus (supported, pendingDelete or pendingDisbable
or 2) SortStatus == Supported
or 3) IsSearchable == true

That would indicate that any SearchParameter with an IsSearchable of true will be included in the job.  But, we don't want to include it unless it is in one of the three states.

Perhaps the flag of IsSearchable should be included together with SortStatus?  I am not certain why this flag is part of the logic.

Reply:
I reverted back to false as it included way more than previously.

Reply:
It is a good point that we may no longer need it. In the event that say a customer upgrades from one FHIR schema to the next and there happens to be search parameters that are now included then they would need to be marked as supported to get reindexed, instead of unsupported.